client/src/commons/Icons/HelpIcon/HelpIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { Tooltip } from '@material-ui/core';
+import HelpOutline from '@material-ui/icons/HelpOutline';
+
+import useStyles from './helpIconStyles';
+
+interface Props {
+    title: string;
+    isWarning: boolean;
+}
+
+const RefreshIcon = (props : Props) => {
```

Comment:
name should match component & file name...

Reply:
whoops , fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1198/commits/b438990cc39295b26e9886a0be75530d913ba2c9)